app/views/Mails/acceptrefuse/sendProposalRefused.scala.html

```diff
@@ -4,28 +4,24 @@
-    <p>Nous vous remercions cependant pour le temps que vous avez consacré à préparer votre sujet.</p>
-    <p>Si vous souhaitez vous inscrire, la conférence étant complète, nous avons mis en place un système d'inscription limité à une place maximum
-        sur le site de la conférence Devoxx France : <a href="https://reg.devoxx.fr/speakers/welcome/@proposal.id/@email">https://reg.devoxx.fr/speakers/welcome/@proposal.id/@email</a></p>
-    <p>Si par ailleurs vous avez eu la chance d'avoir un autre sujet sélectionné et que ce sujet vous donne un badge speaker : ne partagez pas ce lien svp. Il est lié à votre
-        présentation et il ne permet pas l'inscription d'autres personnes.</p>
+    <p></p>
+    <p>Nous avons le regret de vous informer que votre proposition <strong>@proposal.title</strong> (@Messages(proposal.talkType.id)) n'a pas été sélectionnée.</p>
+    <p>Nous vous remercions pour le temps que vous avez consacré à préparer votre sujet.</p>
+    <p></p>
+    <p>Si vous souhaitez vous inscrire, la conférence étant complète, nous avons mis en place un système d'inscription limité à une place maximum sur le site de la conférence Devoxx France : <a href="https://reg.devoxx.fr/speakers/welcome/@proposal.id/@email">https://reg.devoxx.fr/speakers/welcome/@proposal.id/@email</a></p>
+    <p>Si par ailleurs vous avez eu la chance d'avoir un autre sujet sélectionné et que ce sujet vous donne un badge speaker : ne partagez pas ce lien svp. Il est lié à votre présentation et il ne permet pas l'inscription d'autres personnes.</p>
     <p>Si vous avez besoin d'acheter plus d'une place, merci de nous contacter par email: <a href="mailto:info@@devoxx.fr">info@@devoxx.fr</a></p>
-    <p>La sélection cette année a encore été difficile avec près de 800 propositions pour 200 sessions.</p>
     <p></p>
     <p>Merci pour votre proposition,</p>
-    <p>L'équipe de sélection de la conférence @Messages("longYearlyName")</p>
+    <p>L'équipe @Messages("longYearlyName")</p>
     <hr>
 }
-<p>Dear Devoxxian,</p>
-<p>We're sorry to inform you that your proposal "<strong>@proposal.title</strong>" for @Messages("longYearlyName") was <strong>refused</strong>.</p>
-<p>Do keep in mind that we received close to 800 proposals and the available slots are very limited!</p>
+<p>Hi,</p>
 <p></p>
-<p>Proposal : <strong>@proposal.title</strong> [<strong>@Messages(proposal.talkType.id)</strong>]</p>
+<p>We're sorry to inform you that your proposal <strong>@proposal.title</strong> (@Messages(proposal.talkType.id)) for @Messages("longYearlyName") was <strong>refused</strong>.</p>
 <p></p>
 <p>We would like to thank you sincerely for your proposal and shown interest.</p>
-<p>Hopefully we can still welcome you at @Messages("longYearlyName") we set-up a special backdoor for refused proposals, so that you can register :</p>
+<p>Hopefully we can still welcome you at @Messages("longYearlyName") we set up a special service for refused proposals, so that you can register :</p>
```

Comment:
```suggestion
<p>Hopefully we can still welcome you at @Messages("longYearlyName") we have set up a special service for rejected proposals, so that you can register :</p>
```